config/routes.rb

```diff
@@ -46,12 +46,15 @@
     resources :memberships, only: [:create]
     resources :integrations, only: [:index, :create, :destroy]
     resources :changesets, only: [:index]
-    resources :stories, only: [:index, :create, :update, :destroy, :show] do
+    
+    resources :stories, only: [:index, :create, :destroy, :show]  do
```

Comment:
I think this was a bad move, IMO

I wouldn't touch this line in favor of adding a previous route before, outside this block, something like:
```ruby
put 'stories/sort', to: 'stories#sort'
resources :stories, only: [:index, :create, :update, :destroy, :show] do
  ...
end
```